build-system/global-configs/client-side-experiments-config.json

```diff
@@ -26,10 +26,12 @@
     },
     {
       "name": "story-ad-page-outlink",
-      "percentage": 1,
+      "percentage": 0.02,
       "rtvPrefix": [
         "00",
```

Comment:
If you just want it to run everywhere, remove the "rtvPrefix" key - this way it'll be a catch-all